release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -570,6 +570,12 @@ Beginning with {product-title} {product-version}, you can upgrade or downgrade t
 
 In {product-title} {product-version} and later, you can edit the baseboard management controller (BMC) address in the `BareMetalHost` resource of a bare-metal node. The node must be in the `Provisioned`, `ExternallyProvisioned`, `Registering`, or `Available` state. Editing the BMC address in the `BareMetalHost` resource will not result in deprovisioning the node. See xref:../post_installation_configuration/bare-metal-configuration.adoc#editing-a-baremetalhost-resource_post-install-bare-metal-configuration[Editing a BareMetalHost resource] for additional details.
 
+[id="ocp-4-16-attaching-a-non-bootable-iso_{context}"]
+==== Attaching a non-bootable ISO
+
+In {product-title} {product-version} and later, you can attach a generic, non-bootable ISO virtual media image to a provisioned node by using the `DataImage` resource. After you apply the resource, the ISO image becomes accessible to the operating system after it has booted. This is useful for configuring a node after provisioning the operating system and before the node boots for the first time.
```

Comment:
nit : the ISO image becomes accessible to the operating system `on the next reboot`
Please remove the line - "This is useful for configuring a node after provisioning the operating system and before the node boots for the first time."

Reply:
Done